File: mario.py
from cs50 import get_int


def main():
    while True:
        h = get_int("Height: ")
        if h > 0 and h <= 8:
            break

    for i in range(1, h + 1):
        print((h-i)*" " + i*"#")


if __name__ == "__main__":
    main()

File: mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    // Get integer between 1 and 8
    int height;

    do
    {
        height = get_int("Height: ");
    }
    while (height>8||height<1);

    // Create same number of rows as height
    for (int i = 0; i < height; i++)
    {
        // Print " "
        for (int n = height - 1; n > i; n--)
        {
            printf(" ");
        }

        // Print '#'
        for (int h = -1; h < i; h++)
        {
            printf("#");
        }

        // Take new line
        printf("\n");
    }
}
